src/components/bike/BikePrice.vue

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+import { defineComponent, type PropType } from 'vue'
+
+import type { CurrencyCode } from '@/core/config'
+
+export type PropRate = 'daily' | 'weekly' | 'monthly' | 'anually' | 'none'
+
+export default defineComponent({
+  name: 'BikePrice',
+  props: {
+    price: {
+      type: Number,
+      required: true
+    },
+    currency: {
+      type: String as PropType<CurrencyCode>,
+      required: true
+    },
+    rate: {
+      type: String as PropType<PropRate>,
+      default: 'nome'
+    }
+  },
+  computed: {
+    formattedPrice() {
```

Comment:
We have some similar code in `BookingPricing.vue` as well. What do you think about moving it to a utils file, like `prices.ts`, and create a helper function?